bin/masspay.py

```diff
@@ -193,6 +193,11 @@ def post_back_to_gratipay():
     except KeyError:
         gratipay_base_url = 'https://gratipay.com'
 
+    nmasspays = int(requests.get(gratipay_base_url + '/dashboard/nmasspays').text())
+    if nmasspays < 10:
```

Comment:
Hmm ... theoretically, I suppose. We haven't been below 10 in a long time that I can recall (famous last words). This the test I came up with to decide whether masspay had been run the previous week or not. I would say in the case where it _was_ run but there were less than 10 payouts, the person running payday could adapt to the situation by manually modifying the script before rerunning. That should be a rare occurrence.